common-api/src/main/resources/common_api.yaml

```diff
@@ -146,6 +146,16 @@ definitions:
         type: integer
         format: int32
 
+  FeaturedWorkspace:
```

Comment:
You added this to both YAML files, it should only be in one place. workbench.yaml can reference models defined in this file

Reply:
👍